packages/react-components/src/Concepts/Icons/ReactIconGrid.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import { FluentIconsProps } from '@fluentui/react-icons';
+import { SearchBox } from '@fluentui/react/lib/SearchBox';
+import { ChoiceGroup, IChoiceGroupOption, IChoiceGroupStyles } from '@fluentui/react/lib/ChoiceGroup';
+import * as ReactIcons from '@fluentui/react-icons';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useStyles = makeStyles({
+  grid: theme => ({
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'flex-start',
+
+    'grid-list': {
+      alignItems: 'center',
+      color: theme.colorNeutralForeground1,
+      display: 'flex',
+      flexDirection: 'column',
+      fontSize: theme.fontSizeBase600,
+      height: '80px',
+      justifyContent: 'space-around',
+      padding: '8px',
+      width: '80px',
+      overflow: 'hidden',
+
+      'list-span': {
+        color: theme.colorNeutralForeground1,
+        fontSize: theme.fontSizeBase300,
+        opacity: '0',
+      },
+
+      '&:hover': {
+        overflow: 'visible',
+
+        span: {
+          opacity: '1',
+        },
+      },
+    },
+  }),
+
+  searchBox: () => ({
```

Comment:
I think we're working on removing function support from make-styles, so this should just be an object.